src/modules/Topics/components/CreateTopicWizard/CreateTopicWizard.tsx

```diff
@@ -98,6 +101,10 @@ export const CreateTopicWizard: React.FC<CreateTopicWizardProps> = ({
     )
       .createTopic(topic)
       .then(() => {
+        analytics.track('rhosak-create-topic-success', {
```

Comment:
The event name should be in `Upper Case` like .e.g. `RHOSAK Create Topic`.  Also it would be better to put the success/failure designator as a property, so that both outcomes would trigger an event with the same name, but different outcome/result/... property value.